CHANGES.rst

```diff
@@ -1,3 +1,18 @@
+1.1.5 (2023-05-31)
+------------------
+
+New Features
+^^^^^^^^^^^^
+
+Other Changes and Additions
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
++ Most functions are now linked to the documentation (some still need doctrings).
```

Comment:
I know none of the other changelog entries have this but issue or PR numbers should be included, ideally.

```suggestion
+ Most functions are now linked to the documentation (some still need docstrings). [#92]
```